frontend/src/libs/components/checkbox/checkbox.tsx

```diff
@@ -1,20 +1,46 @@
+import { getValidClassNames } from '#libs/helpers/helpers.js';
+
 import styles from './styles.module.scss';
 
 type Properties = {
   label: string;
+  checked?: boolean;
+  name?: string;
   onChange: () => void;
+  disableDefaultStyles?: boolean;
 };
 
-const Checkbox: React.FC<Properties> = ({ label, onChange }) => {
+const Checkbox: React.FC<Properties> = ({
+  label,
+  checked,
+  name,
+  onChange,
+  disableDefaultStyles,
```

Comment:
```suggestion
  isDefaultStylesDisabled,
```
Lets follow quality criteria